### DISCUSSION TITLE
Difference between SW Version 3Q0907530 AD AA L etc. dataset

### DISCUSSION THREAD

**Discussion by deppdive** [jilleb/MQB-FPA#29] [Q&A]
Hello,

there are different Software Releases on the 3Q0907530** box.

I have read the current script supports only the AD Version 3Q0907530**AD**

My car has the AA Build  3Q0907530**AA**

Can I use the script with AA Builds or is there something which needs to be modified?

// Purpose:     VAG MQB FPA (fahrprofilauswahl / driving profile) dataset checksum correcting and saving to "0x00," format
//              Will output to either VCP or ODIS XML format.
//              **Currently only useful for 3Q0907530AD FPA datasets.**

**Comment by batal131** [comment]
I'm using script for AA and AC.
From script result (generated file) you actually need only dataset itself. I'm copying it (data only) to my original dataset file to write it via odis. 
Dataset itself is similar for all AA+ gateways. 

And you can use template without script. Just replace hash manually and copy data to your original file.

**Comment by deppdive** [comment]
Thank you!
 :-) fingers crossed
not sure what happens if there is a cut and paste error, Im using VCS
hopefully the STG19 rejects the update if file does not match the expected specs and restarts with old setting

**Comment by andy0172** [comment]
For the new EV Gateway version 3Q0907530AH its not working, I tried to save FPA after restart, but its not working. Any idea? I can share the dataset if you want

**Reply by djskual** [reply]
hello, i'm looking about AD or AH frimware. Perhaps you hown this? or you know someone? thx
